<?xml version="1.0" encoding="utf-8"?>
<launch>
	<include file="$(find easy_handeye)/launch/calibrate.launch">
		<arg name="eye_on_hand" value="false"/>
		<arg name="namespace_prefix" value="kalibracja"/>

		<!-- Układ bazowy robota i układ efektora-->
		<arg name="robot_base_frame" value="sl_base"/>
		<arg name="robot_effector_frame" value="effector_frame"/>

		<!-- Układ kamery i układ markera -->
		<arg name="tracking_base_frame" value="kinect2_rgb_optical_frame"/>
		<arg name="tracking_marker_frame" value="ar_marker_2"/>

		<arg name="freehand_robot_movement" value="true" />
		<arg name="publish_dummy" value="false" />
		<arg name="start_rviz" value="false" />	
  	</include>
</launch>